packages/test/test-end-to-end-tests/src/test/cellEndToEndTests.spec.ts

```diff
@@ -231,6 +231,24 @@ describeFullCompat("SharedCell", (getTestObjectProvider) => {
         verifyCellValue(await getCellDataStore(getCellDataStore(Promise.resolve(sharedCell2))), cellValue, 2);
         verifyCellValue(await getCellDataStore(getCellDataStore(Promise.resolve(sharedCell3))), cellValue, 3);
     });
+
+    it("attaches if referring SharedCell becomes attached or is already attached", async () => {
+        const detachedCell1: ISharedCell = SharedCell.create(dataObject1.runtime);
+        const detachedCell2: ISharedCell = SharedCell.create(dataObject1.runtime);
+
+        // When an unattached cell refers to another unattached cell, both remain unattached
+        detachedCell1.set(detachedCell2.handle);
+        assert.equal(sharedCell1.isAttached(), true, "sharedCell1 is not attached");
+        assert.equal(detachedCell1.isAttached(), false, "detachedCell1 is not attached");
+        assert.equal(detachedCell2.isAttached(), false, "detachedCell2 is not attached");
```

Comment:
Remove "not" from the message in the above two lines.

Reply:
Corrected!

Reply:
Same for other tests